runtime/compiler/arm/codegen/J9AheadOfTimeCompile.cpp

```diff
@@ -378,26 +377,18 @@ uint8_t *J9::ARM::AheadOfTimeCompile::initializeAOTRelocationHeader(TR::Iterated
          }
          break;
 
-      case TR_InlinedStaticMethodWithNopGuard:
-      case TR_InlinedSpecialMethodWithNopGuard:
-      case TR_InlinedVirtualMethodWithNopGuard:
       case TR_InlinedVirtualMethod:
-      case TR_InlinedInterfaceMethodWithNopGuard:
       case TR_InlinedInterfaceMethod:
-      case TR_InlinedHCRMethod:
          {
          guard = (TR_VirtualGuard *) relocation->getTargetAddress2();
 
-         if (relocation->getTargetKind() == TR_InlinedHCRMethod)
```

Comment:
good opportunity to fix ARM to make it consistent with other platforms! This work is really showing its value :) .

However, I'd like to see this change made under a separate (probably prequel would be easiest) commit, if possible, since it represents a change in behaviour not  connected to the consolidation effort.